routes/airport.js

```diff
@@ -0,0 +1,138 @@
+var express = require("express");
+var app = express();
+var async = require("async");
+const fs = require("fs");
+
+var data = fs.readFileSync("./airport.json", "utf-8");
+var airportData = JSON.parse(data);
+
+function calcCrow(lat1, lon1, lat2, lon2) {
+  var R = 6371; // km
+  var dLat = toRad(lat2 - lat1);
+  var dLon = toRad(lon2 - lon1);
+  var lat1 = toRad(lat1);
+  var lat2 = toRad(lat2);
+
+  var a =
+    Math.sin(dLat / 2) * Math.sin(dLat / 2) +
+    Math.sin(dLon / 2) * Math.sin(dLon / 2) * Math.cos(lat1) * Math.cos(lat2);
+  var c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
+  var d = R * c;
+  return d;
+}
+// Converts numeric degrees to radians
+function toRad(Value) {
+  return (Value * Math.PI) / 180;
+}
+
+app.get("/list", function (req, res) {
+  try {
+    if (data) {
+      var airData = [];
+      for (var i = 0; i < airportData.length; i++) {
+        airData.push({ locationId: airportData[i] });
+      }
```

Comment:
no need to loop the data,
as you have airport data already

Reply:
removed